src/components/table/__spec__.js

```diff
@@ -1019,6 +1019,23 @@ describe('Table', () => {
     });
   });
 
+  describe('isPassive', () => {
```

Comment:
Is it worth adding a case covering when both `highlightable` and `selectable` are true, or is that an invalid scenario?

Reply:
No good point I'lll add for the sake of completeness 👍